### PR TITLE
Fixes #53 : Update the readme to reflect the use of devcontainer

### LINKED ISSUE #53: Update the readme to reflect the use of devcontainer
### Description

Currently the README mentions the use of virtual envs which are no longer necessary when using a devcontainer.

### PR DESCRIPTION
**Pull request by redat97**
Fixes #53

### CODE REVIEW

**Review by k-allagbe — CHANGES_REQUESTED** [review]
I would suggest to restructure the README like this:

1. Setting up the development environment
    - Using devcontainer (already setup in devcontainer.json, all the developer needs to do is open the project in the devcontainer)
    - Using virtual environment (I would suggest here to link to an external guide as this is well known already)
2. Environment variable configuration
3. Running the app locally
4. Running the app from dockerfile

**Review by rngadam — CHANGES_REQUESTED** [review]
your commits don't reference the issue

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Follow the instructions below to set up a Quart application in your environment:
```
Comment (rngadam) [inline README.md:5]: @k-allagbe I find the README.md extremely confusing. I suggest we break it down into separate docs to have a short and clean README.md. Should be done in a separate issue/PR

**Review by rngadam — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (k-allagbe) [reply README.md:5]: Agreed. https://github.com/ai-cfia/membrane-backend/issues/83

**Comment by rngadam** [comment]
> I would suggest to restructure the README like this:
> 
> 1. Setting up the development environment
>    
>    * Using devcontainer (already setup in devcontainer.json, all the developer needs to do is open the project in the devcontainer)
>    * Using virtual environment (I would suggest here to link to an external guide as this is well known already)
> 2. Environment variable configuration
> 3. Running the app locally
> 4. Running the app from dockerfile

@k-allagbe 
These details should be in the issue #83

**Comment by k-allagbe** [comment]
> > I would suggest to restructure the README like this:
> > 
> > 1. Setting up the development environment
> >    
> >    * Using devcontainer (already setup in devcontainer.json, all the developer needs to do is open the project in the devcontainer)
> >    * Using virtual environment (I would suggest here to link to an external guide as this is well known already)
> > 2. Environment variable configuration
> > 3. Running the app locally
> > 4. Running the app from dockerfile
> 
> @k-allagbe These details should be in the issue #83

Done

**Review by k-allagbe — APPROVED** [review]
I'll take care of fixing any confusion in the README myself in #83.

**Review by rngadam — APPROVED** [review]
don't manually number heading. either use the automatic numbering or don't number

**Pull request merged**